pages/[username]/[collectionId]/[nftId]/index.tsx

```diff
@@ -16,8 +18,34 @@ export default function NftDetailPage({ nftId }: NftDetailPageProps) {
   return <GalleryRoute element={<NftDetailPageScene nftId={nftId} />} />;
 }
 
-export const getServerSideProps: GetServerSideProps<NftDetailPageProps> = async ({ params }) => ({
-  props: {
-    nftId: params?.nftId ? (params.nftId as string) : undefined,
-  },
-});
+export const getServerSideProps: GetServerSideProps<NftDetailPageProps> = async ({ params }) => {
+  const username = params?.username ? (params.username as string) : undefined;
+  const nftId = params?.nftId ? (params.nftId as string) : undefined;
+  return {
```

Comment:
could we temporarily disable this in prod?

we usually use [featureFlag.ts](https://github.com/gallery-so/gallery/blob/main/src/utils/featureFlag.ts) but it relies on `window`... is it possible to detect the origin in `getServerSideProps`?

Reply:
ahh we can use `NEXT_PUBLIC_VERCEL_ENV`

Reply:
+1 for hiding on prod for now